docs/digital_me/getting_started.md

```diff
@@ -302,7 +302,7 @@ zos_client.ping()
 
 Get the SAL interface of your ThreeFold node and list the containers, you should see the `zrobot` container:
 ```python
-zos_sal = j.clients.zos.sal.get_node(instance=zos_instance_name)
+zos_sal = j.sal_zos.node.get(instance=zos_instance_name)
```

Comment:
`j.sal_zos.node.get` accept a zos client as argument not an instance name.
To get the same as `j.clients.zos.sal.get_node` now you can just do `j.clients.zos.get(instance)`